x/tokenomics/keeper/settle_pending_claims_test.go

```diff
@@ -158,55 +112,139 @@ func (s *TestSuite) SetupTest() {
 	))
 	require.NoError(t, err)
 
-	// Construct a valid session tree with 100 relays.
-	s.numRelays = uint64(100)
-	sessionTree := testtree.NewFilledSessionTree(
-		sdkCtx, t,
-		s.numRelays, service.ComputeUnitsPerRelay,
-		"supplier", supplierOwnerAddr,
-		sessionHeader, sessionHeader, sessionHeader,
-		keyRing,
-		ringClient,
-	)
-
-	// Calculate the number of claimed compute units.
-	s.numClaimedComputeUnits = s.numRelays * service.ComputeUnitsPerRelay
-
-	targetNumRelays := s.keepers.ServiceKeeper.GetParams(sdkCtx).TargetNumRelays
-	s.relayMiningDifficulty = servicekeeper.NewDefaultRelayMiningDifficulty(
-		sdkCtx,
-		s.keepers.Logger(),
-		testServiceId,
-		targetNumRelays,
-		targetNumRelays,
-	)
-
-	// Calculate the number of estimated compute units.
-	s.numEstimatedComputeUnits = getEstimatedComputeUnits(s.numClaimedComputeUnits, s.relayMiningDifficulty)
+	appStake := types.NewCoin("upokt", math.NewInt(1000000))
 
-	// Calculate the claimed amount in uPOKT.
-	sharedParams := s.keepers.SharedKeeper.GetParams(sdkCtx)
-	s.claimedUpokt = getClaimedUpokt(sharedParams, s.numEstimatedComputeUnits, s.relayMiningDifficulty)
+	// Setup the test for each service:
+	// - Create and store the service in the service keeper.
+	// - Create and store an application staked to the service.
+	// - Create a supplier service config for the service.
+	supplierServiceConfigs := make([]*sharedtypes.SupplierServiceConfig, 0, len(testServiceIds))
+	appAddresses := make([]string, 0, len(testServiceIds))
+	for i, serviceId := range testServiceIds {
+		service := sharedtypes.Service{
```

Comment:
For this function and the one below (for claims and proofs), can you move it out into a helper?

The test is become hard to read.